Sources/TensorFlow/Operators/Math.swift

```diff
@@ -2632,7 +2664,6 @@ public extension Tensor where Scalar: TensorFlowFloatingPoint {
         logSumExp(alongAxes: axes)
     }
 }
-
```

Comment:
Please re-add this deleted line. There should be one empty line between declarations.
When you're unsure about formatting rules, you can use [`swift-format`](https://github.com/apple/swift-format) to auto-format newly added code:
```console
$ swift-format Sources/TensorFlow/Operators/Math.swift
# Project-specific configuration file: https://github.com/tensorflow/swift-apis/blob/master/.swift-format
```